azure-pipelines.yml

```diff
@@ -1,6 +1,15 @@
 trigger:
 - master
 
+schedules:
+- cron: "0 23 * * *"
+  displayName: Nightly
+  always: false
+  branches:
+    include:
+    - master
+    - 0.53_maintenance
```

Comment:
An explicit list of maintenance branches is going to get out of sync rapidly. I suggest we just remove this, because we always develop on master, and backports to maintenance are usually followed immediately by a release anyway.

Reply:
Fixed in https://github.com/GafferHQ/gaffer/pull/3248/commits/0125d9ff8321f0b11ce9f70ae0b9713b5db6cb4f